backend/typescript/middlewares/validators/util.ts

```diff
@@ -34,6 +50,28 @@ export const validatePrimitive = (value: any, type: Type): boolean => {
   }
 };
 
+export const validateArrayOfObjects = (
```

Comment:
hmm i feel like it might be easier to just have a function which validates a specific clause object (e.g. `validateWaiverClause`) and call that function on every clause in the waiver? the combination of this with `getType` seems a bit overcomplicated? and not very efficient since we need to getType for each key for every object. does anyone else have thoughts on thiss @uwblueprint/focus-on-nature